bokeh/models/glyphs.py

```diff
@@ -164,6 +164,11 @@ class Marker(XYGlyph, LineGlyph, FillGlyph):
     The y-axis coordinates for the center of the markers.
     """)
 
+    hit_radius = ScreenDistanceSpec(default=None, help="""
```

Comment:
The "spec" types are specifically for mediating thigns that can either be a single scalar value *or* a reference to an entire column of data in a CDS. This is always just a number, so it should be `Float`